src/app.js

```diff
@@ -1,18 +1,30 @@
 import {run} from '@cycle/core';
 import CycleDOM from '@cycle/dom';
-import {Observable} from 'rx'
-import Todos from './components/Todos/index';
-// import CustomDrivers from './drivers';
+import {Observable} from 'rx';
 import storageDriver from '@cycle/storage';
-import todos from './components/todos/index';
+// THE MAIN FUNCTION
+// This is the todo list component.
+import Todos from './components/Todos/index';
 
 const main = Todos;
 
+// THE ENTRY POINT
+// This is where the whole story starts.
+// `run` receives a main function and an object with
+// the source streams.
 run(main, {
+  // THE DOM DRIVER
+  // We create a DOM driver by passing
+  // a DOM selector to the `makeDOMDriver` function.
```

Comment:
The comment is just paraphrasing the code, so it's not useful. Either remove the comment or explain the purpose of passing a selector to makeDOMDriver.